src/components/WebcamCapture/WebcamCaptureContainer.js

```diff
@@ -10,22 +14,43 @@ class WebcamCaptureContainer extends Component {
 	constructor(props) {
 		super(props);
 		this.webcam = React.createRef();
-		this.capture = this.capture.bind(this);
-	}
-
-	capture() {
-		return this.webcam.current.getScreenshot();
+		this.model = new Model();
+		this.model.load();
 	}
 
 	componentDidMount() {
 		if (navigator.mediaDevices) {
 			navigator.mediaDevices
 				.getUserMedia({video: true})
-				.then(connected => this.setState({cameraConnected: true}))
-				.catch(err => this.setState({cameraConnected: false}));
+				.catch(err => this.setState({cameraConnected: false}))
```

Comment:
can you just you empty input arrow function here and below like `() => `
variable `err` is not used?

Reply:
I agree, that was still there from previous commits.